apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.RetryConformanceTests/RetryConformanceTest.cs

```diff
@@ -122,6 +137,7 @@ bool InstructionContainsErrorCode(HttpStatusCode statusCode)
     private async Task<TestResponse> CreateRetryTestResourceAsync(InstructionList instructionList, Method method)
     {
         var stringContent = GetBodyContent(method.Name, instructionList);
+        Console.WriteLine("Creating the resource for method: " + method.Name + " for instructions: " + instructionList.Instructions.ToString()+" URI: "+ _fixture.HttpClient.BaseAddress.ToString());
```

Comment:
We can try the following 2 things as the issue seems to be in Storage Test Bench:
1. Try and add this in the test startup AppContext.SetSwitch("System.Net.Http.SocketsHttpHandler.Http2UnencryptedSupport", true);

2. Try and add delay (may be a 2-3 second delay, so that Storage Test Bench gets time to process the request completely)

Reply:
Ahh.. Still the same error -_- :(
Thanks for the options though